js/gallery.js

```diff
@@ -0,0 +1,184 @@
+;(function($){
```

Comment:
@samipshrestha you can reformat your code with respect to following assumption.
1) Proper documentation for the function.
2) Remove unnecessary spaces and remove console and debugger kinds of stuff.
3) Move all the function to the bottom. Code starting with your plugin name would be better one.